NEWS.md

```diff
@@ -3,6 +3,12 @@
 ### Major breaking change
 * `chunks` have now been deprecated and will be removed from the package in a future release. The new `qenv` object should be used instead. See the new `qenv` vignette in the package for further details.
 
+### Enhancements
+* Improved output format of warnings in `get_warnings`.
```

Comment:
qenv hasn't been released yet, so improvement of `get_warnings` is not an item for a log change
```suggestion
```